### PR TITLE
docs: add semi relative time ranges

### PR DESCRIPTION
**Pull request by imatwawana**
Adds information about semi-relative time ranges
Other small copy/formatting fixes

### CODE REVIEW

File: docs/sources/dashboards/use-dashboards/index.md [hunk 1]
```diff
@@ -156,6 +158,32 @@ You can set an absolute time range in the following ways:
...
 
+#### Semi-relative time range
+
+{{% admonition type="Note" %}}
+
+Grafana Alerting does not support semi-relative time ranges.
+
+{{% /admonition %}}
+
+You can also use the absolute time range settings to set a semi-relative time range. Semi-relative time range dashboards are useful when you need to monitor the progress of something over time, but you also want to see the entire history from a beginning point in time.
```
Comment (imatwawana) [inline docs/sources/dashboards/use-dashboards/index.md:169]:
```suggestion
You can also use the absolute time range settings to set a semi-relative time range. Semi-relative time range dashboards are useful when you need to monitor the progress of something over time, but you also want to see the entire history from a starting point.
```

File: docs/sources/dashboards/use-dashboards/index.md [hunk 2]
```diff
@@ -105,13 +105,15 @@ The following table provides example relative ranges:
...
 | Previous fiscal year   | `now-1y/fy` | `now-1y/fy` |
 
-#### Note about Grafana Alerting
+{{% admonition type="Note" %}}
 
-For Grafana Alerting, we do not support the following syntaxes at this time.
+Grafana Alerting does not support the following syntaxes at this time:
 
 - now+n for future timestamps.
 - now-1n/n for "start of n until end of n" because this is an absolute timestamp.
```
Comment (imatwawana) [inline docs/sources/dashboards/use-dashboards/index.md:113]: @brendamuir - I've added a note in the semi-relative time range section about it not being supported in Alerting, but this note was already here and I just wanted to check with you that it's still valid.

**Review by adeverteuil — APPROVED** [review]
Nice :) thanks for adding this to the docs!

**Comment by imatwawana** [comment]
> Nice :) thanks for adding this to the docs!

no problem! @brendamuir was the eagle eye who caught that this should be added.

**Review by brendamuir — APPROVED** [review]

**Pull request merged**